### PR TITLE
Review Suggestions on #9

### PR DESCRIPTION
**Pull request by rishiip**
Ref PR - #9

### CODE REVIEW

**Comment by yedhink** [comment]
I will review this PR once https://github.com/rishiip/wheel/pull/9#discussion_r949862101 is answered.

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 1]
```diff
@@ -1,11 +1,9 @@
 import React from "react";
 
-import dayjs from "dayjs";
-import relativeTime from "dayjs/plugin/relativeTime";
 import { MenuVertical, Clock } from "neetoicons";
 import { Typography, Dropdown, Tag, Tooltip, Avatar } from "neetoui";
 
-dayjs.extend(relativeTime);
+import dayjs from "../../../lib/dayjs";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:6]: - First thing, if the relative path becomes too long, then it's better to create an webpack alias for it. Please read: https://www.bigbinary.com/books/learn-rubyonrails-book/setting-up-react-environment#aliases 
- Secondly over here you don't need to import from `lib/dayjs`. That's same as your previous implementation where you had imported the dayjs instance from `constants`. Here import directly from `dayjs` itself. But in the root component, something like `Main.jsx`, add a single import, like `import "lib/dayjs"` for initialization purposes. That's good enough I believe.

File: app/javascript/src/lib/dayjs.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+dayjs.extend(relativeTime);
+
+export default dayjs;
```
Comment (yedhink) [inline app/javascript/src/lib/dayjs.js:6]: You don't have to export this instance. `dayjs` instance is global!

**Review by yedhink — CHANGES_REQUESTED** [review]
@rishiip Once the following changes are made, please go ahead and do `Squash and Merge`.

File: app/javascript/src/components/Dashboard/Notes/Note.jsx (hunk 1 above)
Comment (rishiip) [reply app/javascript/src/components/Dashboard/Notes/Note.jsx]: @yedhink - This is done. Thanks a ton for helping me here 🙌🏼

File: app/javascript/src/lib/dayjs.js (hunk 2 above)
Comment (rishiip) [reply app/javascript/src/lib/dayjs.js]: This is done. Have a look at latest code.

**Comment by yedhink** [comment]
@rishiip I am merging this PR now. But moving forward, please read the comments carefully: https://github.com/rishiip/wheel/pull/12#pullrequestreview-1080360066

image

**Pull request merged**